manifests/compute/foreman_compute.pp

```diff
@@ -1,7 +1,7 @@
-# Provides a common class to install foreman-compute
-# used by multiple compute resources
+# = Provides a common class to install foreman-compute
+#   used by multiple compute resources
```

Comment:
Can you use multi line statements like this? I wonder if the various parsers we apply actually understand this.